### PR TITLE
Fix/world cup database

### PR DESCRIPTION
**Pull request by RafaelDavisH**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)

Fix typos and rephrase some instructions in TUTORIAL.md

### CODE REVIEW

File: tutorial.json [hunk 1]
```diff
@@ -61,12 +58,10 @@
...
-            "You should correctly complete the queries in the `queries.sh` file. The first one is completed for you. Fill in each empty `echo` command to get the output of what is suggested with the command above it. Only use a single line like I have demonstrated in the first query. The output should match what is in the `example_output.txt` file. Take note of the order of some of the results"
+            "You should correctly complete the queries in the `queries.sh` file. The first one is completed for you. Fill in each empty `echo` command to get the output of what is suggested with the command above it. Only use a single line, as I have demonstrated in the first query. The output should match what is in the `example_output.txt` file. Take note of the order of some of the results"
           ],
           "setup": {
-            "commits": [
-              "76d45ba9f98ce4fa02459c14751898605abaadff"
-            ]
+            "commits": []
```
[the remark this answers is not in the document]
Comment (moT01) [reply tutorial.json:64]: Yea, I intend to change the changelog of all the tutorials to just give in a link to the contributing docs. You will need to create that branch before you run `coderoad build` @RafaelDavisH.

[the remark this answers is not in the document]
Comment (ieahleen) [reply tutorial.json:64]: I can help with that - the changelog file doesn't need special workflows, right?

[the remark this answers is not in the document]
Comment (moT01) [reply tutorial.json:64]: nope, a simple PR should work @ieahleen

[the remark this answers is not in the document]
Comment (RafaelDavisH) [reply tutorial.json:64]: > Yea, I intend to change the changelog of all the tutorials to just give in a link to the contributing docs. You will need to create that branch before you run `coderoad build` @RafaelDavisH.

What branch do I need to create? 
I created the 'fix/world-cup-database`
then made the changes, staged and commit them 
ran `coderoad build` then staged and commit those changes too
made PR

or was I supposed to create v1.0.2 branch?

[the remark this answers is not in the document]
Comment (moT01) [reply tutorial.json:64]: you need to create the v1.0.1 branch @RafaelDavisH

☝️ https://github.com/freeCodeCamp/learn-world-cup-database/pull/7#discussion_r739372713

Let me know if you want to pair up for a bit, I can help direct you if you want.

[the remark this answers is not in the document]
Comment (RafaelDavisH) [reply tutorial.json:64]: Thanks @moT01. I take your offer for pairing. I'm afraid this thread could get longer if I keep trying.

**Pull request closed without merging**